CHANGELOG.latest.md

```diff
@@ -1,5 +1,4 @@
-### API Updates
-* `Transaction` class properties `revenuecatId` and `productId` have been deprecated. Use `transactionIdentifier` and `productIdentifier` respectively instead (#587)
-
 ### Other Changes
-* Remove billing client 3 support (#567) via beylmk (@beylmk)
+* CleverTapID integration (#593) via Joshua Liebowitz (@taquitos)
+* Rename package name of purchase tester app to com.revenuecat.purchases_sample (#592) via Toni Rico (@tonidero)
+* Fix prepare next version (#591) via Toni Rico (@tonidero)
```

Comment:
I would probably remove this one from the changelog? I don't consider it user-facing

Reply:
👍